rules/order/src/PropertyRanker.php

```diff
@@ -72,4 +66,30 @@ public function rank(Property $property): int
 
         throw new NotImplementedException(get_class($varType));
     }
+
+    private function isFive(Type $type): bool
+    {
+        if ($type instanceof StringType) {
+            return true;
+        }
+
+        if ($type instanceof IntegerType) {
+            return true;
+        }
+
+        if ($type instanceof BooleanType) {
+            return true;
+        }
```

Comment:
This should be refactored into array map constant, e.g.:

```php
private const TYPE_TO_RANK = [
    StringType::class => 5,
    // ...
    ArrayType::class => 10,
];
```

Then foreached.

Reply:
implemented.